content/providers/01-ai-sdk-providers/22-grok.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Grok
+description: Learn how to use Grok.
+---
```

Comment:
- need to remove the other x.ai provider page
- might be useful to say X.ai Grok in the title & below to make it more searchable and to distinguish from Groq

Reply:
They seem to brand themselves as `xAI` at https://x.ai/about and https://x.ai/ so I'll use that. Will rm the other and update as you note here.